### ISSUE TITLE
Difficulty understanding the README

### ISSUE THREAD

**Issue by TheEvilSkeleton** [tkashkin/GameHub#406]
I'm trying to revamp the README to make it easier to understand and more organized. In https://github.com/tkashkin/GameHub/blob/master/README.md#building-1, you will see the following:

#### Building
```bash
git clone https://github.com/tkashkin/GameHub.git
cd GameHub
scripts/build.sh build_deb
```
Can you explain what the script does exactly? It's really hard to understand what it does exactly (I took a look at the script and I have no idea what it does)

**Comment by tkashkin** [comment]
`build_deb` does multiple things, most of these things are only relevant for CI builds:
* removes some package dependencies that are not available on Ubuntu 16.04 if build runs on 16.04
* builds binary .deb package and moves it to the parent directory (`../GameHub-*.deb`)
* if there are encryption keys in the environment, for each target distro (lines 34-35):
  - builds source package
  - signs it
  - uploads source package to Launchpad with `dput`

Basically it just builds a .deb package while not running on AppVeyor.
If GameHub source code is in `~/Projects/GameHub`, it should build `~/Projects/GameHub-$version-amd64.deb`.

**Comment by TheEvilSkeleton** [comment]
Awesome, that helped a lot. Thank you so much

**Comment by TheEvilSkeleton** [comment]
@tkashkin another question, in https://github.com/tkashkin/GameHub/releases, there are `.deb` packages, is it possible to just download and then run `sudo dpkg -i *.deb` to install the package?

**Comment by tkashkin** [comment]
Yes, that's what this script builds.

**Comment by TheEvilSkeleton** [comment]
Alright, thank you so much again
